data_source_obmcs_database_databases_test.go

```diff
@@ -41,38 +83,36 @@ func (s *DatabaseDatabasesTestSuite) TestReadDatabases() {
 			{
 				Config: s.Config + `
 				data "oci_database_db_systems" "t" {
-				  compartment_id = "${var.compartment_id}"
+					compartment_id = "${var.compartment_id}"
 				}
 				data "oci_database_db_homes" "t" {
-				  compartment_id = "${var.compartment_id}"
-				  db_system_id = "${oci_database_db_system.t.id}"
+					compartment_id = "${var.compartment_id}"
+					db_system_id = "ocid1.dbsystem.oc1.phx.abyhqljskxaehm3osd7ekh5sxqdp7y6ob3u54ayo6y57pcvcbk3cf75kd2oa"
```

Comment:
Will this fail if the tests are configured to use a different tenancy and/or different region? Should it be set in an env variable?

Reply:
Good catch, I was running these tests against an existing DBSystem to speed things along, will update